web/styles/pygments.css

```diff
@@ -1,214 +1,475 @@
 } /* Name.Tag */
 .codehilite .nv {
-    color: hsl(241deg 68% 28%);
+    color: #953800;
 } /* Name.Variable */
-.codehilite .nx {
-    color: hsl(0deg 0% 26%);
-} /* Not sure? */
 .codehilite .ow {
-    color: hsl(276deg 100% 56%);
-    font-weight: bold;
+    color: #8250df;
 } /* Operator.Word */
 .codehilite .w {
-    color: hsl(0deg 0% 73%);
+    color: #24292f;
 } /* Text.Whitespace */
+.codehilite .mb {
+    color: #953800;
+} /* Literal.Number.Bin */
 .codehilite .mf {
-    color: hsl(195deg 100% 35%);
+    color: #953800;
 } /* Literal.Number.Float */
 .codehilite .mh {
-    color: hsl(195deg 100% 35%);
+    color: #953800;
 } /* Literal.Number.Hex */
 .codehilite .mi {
-    color: hsl(195deg 100% 35%);
+    color: #953800;
 } /* Literal.Number.Integer */
 .codehilite .mo {
-    color: hsl(195deg 100% 35%);
+    color: #953800;
 } /* Literal.Number.Oct */
+.codehilite .sa {
+    color: #0550ae;
+} /* Literal.String.Affix */
 .codehilite .sb {
-    color: hsl(86deg 57% 40%);
+    color: #0550ae;
 } /* Literal.String.Backtick */
 .codehilite .sc {
-    color: hsl(86deg 57% 40%);
+    color: #0550ae;
 } /* Literal.String.Char */
+.codehilite .dl {
+    color: #0550ae;
+} /* Literal.String.Delimiter */
 .codehilite .sd {
-    color: hsl(86deg 57% 40%);
-    font-style: italic;
+    color: #0550ae;
 } /* Literal.String.Doc */
 .codehilite .s2 {
-    color: hsl(225deg 71% 33%);
+    color: #0550ae;
 } /* Literal.String.Double */
 .codehilite .se {
-    color: hsl(26deg 69% 43%);
-    font-weight: bold;
+    color: #0550ae;
 } /* Literal.String.Escape */
 .codehilite .sh {
-    color: hsl(86deg 57% 40%);
+    color: #0550ae;
 } /* Literal.String.Heredoc */
 .codehilite .si {
-    color: hsl(336deg 38% 56%);
-    font-weight: bold;
+    color: #0550ae;
 } /* Literal.String.Interpol */
 .codehilite .sx {
-    color: hsl(120deg 100% 25%);
+    color: #0550ae;
 } /* Literal.String.Other */
 .codehilite .sr {
-    color: hsl(189deg 54% 49%);
+    color: #0550ae;
 } /* Literal.String.Regex */
 .codehilite .s1 {
-    color: hsl(86deg 57% 40%);
+    color: #0550ae;
 } /* Literal.String.Single */
 .codehilite .ss {
-    color: hsl(241deg 68% 28%);
+    color: #0550ae;
 } /* Literal.String.Symbol */
 .codehilite .bp {
-    color: hsl(120deg 100% 25%);
+    color: #953800;
 } /* Name.Builtin.Pseudo */
+.codehilite .fm {
+    color: #0550ae;
+} /* Name.Function.Magic */
 .codehilite .vc {
-    color: hsl(241deg 68% 28%);
+    color: #953800;
 } /* Name.Variable.Class */
 .codehilite .vg {
-    color: hsl(241deg 68% 28%);
+    color: #953800;
 } /* Name.Variable.Global */
 .codehilite .vi {
-    color: hsl(241deg 68% 28%);
+    color: #953800;
 } /* Name.Variable.Instance */
+.codehilite .vm {
+    color: #953800;
+} /* Name.Variable.Magic */
 .codehilite .il {
-    color: hsl(0deg 0% 40%);
+    color: #953800;
 } /* Literal.Number.Integer.Long */
+.codehilite .highlight {
+    background: #fff;
+    color: #24292f;
+}
+
+.dark-theme {
```

Comment:
This doesn’t select the automatic theme correctly. You need to use the same pattern from the other files:

```css
%dark-theme {
    /* dark colors go here */
}

@media screen {
    :root.dark-theme {
        @extend %dark-theme;
    }
}

@media screen and (prefers-color-scheme: dark) {
    :root.color-scheme-automatic {
        @extend %dark-theme;
    }
}
```

Reply:
Thanks, Anders. I knew I was doing that wrong, but thank you for the correction